client/dist/bundle.js

```diff
@@ -94,7 +94,31 @@
 /***/ (function(module, __webpack_exports__, __webpack_require__) {
```

Comment:
it might look more professional to have your bundle.js be in your .gitignore file

Reply:
That's a great idea! Added ✔️